collections/nemo_nlp/nemo_nlp/data/datasets/bert_pretraining.py

```diff
@@ -263,7 +263,7 @@ def truncate_seq_pair(a, b, max_num_tokens):
 
         # TODO: wrap the return value with () for consistent style.
         return np.array(input_ids), input_type_ids,\
-            np.array(input_mask, dtype=np.float32), np.array(output_ids),\
+            np.array(input_mask, dtype=np.long), np.array(output_ids),\
```

Comment:
will have to update this on all other datasets in nemo_nlp that use bert too

Reply:
done